test/JSON/jx2.js

```diff
@@ -187,7 +187,7 @@ function repf(key, value) {
 
 var objectArray = [
 { obj: "SampleTest", desc: "String simple" },
-{ obj: "/test ze\0ro\vString\n_u4:\u0061_u2:\xbc_u1:\x0e_u2clean:\x8f", desc: "String complex" },
+{ obj: "/test ze\0ro\vString\n_u4:\u0061_u2:\x62_u1:\x0e_u2clean:\x50", desc: "String complex" },

```

Comment:
We were discussing offline and encoding related test updates are removed from the PR (apart from trim.js update). There will be an additional PR targeting this problem in general.

> Was this the only test input with non-ASCII characters (i.e., above 127) in the project being passed to JSON.stringify? If so, this might reduce test coverage.

There are couple of things to note with this test. This test has a platform dependency (timezone) hence it's already excluded from jenkins (already reduced coverage).

So, If upcoming PR may match Windows baseline for this file on xplat, we are good to go. If It doesn't, I would update this file back with ASCII and prepare a separate test file for non ASCII tests without baseline. At least we could test that part on jenkins.